pkg/util/validation.go

```diff
@@ -0,0 +1,8 @@
+package util
+
+// Supported values for ingestion rate strategies
+// TODO: check if these can be moved to dskit
+const (
+	LocalIngestionRateStrategy  = "local"
```

Comment:
these are already defined in `/modules/overrides`. should we use those?

Reply:
totally missed those, yeah makes sense to use those.